### PR TITLE
Update HISTORY.md to mention a bug fix

### PR DESCRIPTION
**Pull request by riversand963**

### CODE REVIEW

**Comment by ajkr** [comment]
It might be helpful to users to describe the impact. Can it cause corruption? If so is it localized to an SST or can it spread to the whole DB (e.g., if LSM invariants are violated)? Would it be a problem with all configurations or only certain ones?

File: HISTORY.md [hunk 1]
```diff
@@ -17,6 +17,7 @@
 * Fix a race condition between WritePrepared::Get and ::Put with duplicate keys.
 * Fix crash when memtable prefix bloom is enabled and read/write a key out of domain of prefix extractor.
 * Adjust WriteBufferManager's dummy entry size to block cache from 1MB to 256KB.
+* Fix a bug in GetOverlappingInputsRangeBinarySearch where a pointer should be dereferenced.
```
Comment (siying) [inline HISTORY.md]: It needs to be clearer. Right now it is not very helpful to users as a part of the release note.

Also, it is under release 6.1.1, which claims to be cut in 4/9/2019. Something must be wrong somewhere.

**Comment by siying** [comment]
Can we merge it?

**Comment by miasantreble** [comment]
@riversand963 ping

**Pull request closed without merging**